cmake/SetupPackages.cmake

```diff
@@ -101,6 +112,7 @@ else()
       CMAKE_ARGS                
           -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
           -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+          -DCMAKE_CXX_COMPILER_ARG1=${CMAKE_CXX_COMPILER_ARG1}
```

Comment:
This allows passing through the compile timing thing that I was using.  It's just a way to pass through something positional to a compiler or compiler wrapper.  It certainly doesn't have to stay in, but I didn't see a reason to keep it out.